spec/actions/versions/create_version_spec.cr

```diff
@@ -0,0 +1,52 @@
+require "../../spec_helper"
+require "../../../src/actions/versions/create_version.cr"
+
```

Comment:
What about we create a file that contains all this helper methods? We are repeating this methods in several files